storage/kv/badger_test.go

```diff
@@ -141,6 +141,11 @@ func TestBadgerDb_GetMany(t *testing.T) {
 	})
 	require.NoError(t, err)
 	require.Equal(t, 4, len(results))
+	err = db.GetMany(prefix, [][]byte{}, func(obj basedb.Obj) error {
```

Comment:
This would be a bit more readable with a newline
```suggestion
	
	err = db.GetMany(prefix, [][]byte{}, func(obj basedb.Obj) error {
```